src/sentry/migrations/0267_sentry_release_version_btree_idx.py

```diff
@@ -0,0 +1,39 @@
+# Generated by Django 2.2.24 on 2022-01-21 09:04
+
+from django.db import migrations, models
+
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production. For
+    # the most part, this should only be used for operations where it's safe to run the migration
+    # after your code has deployed. So this should not be used for most operations that alter the
+    # schema of a table.
+    # Here are some things that make sense to mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that they can
+    #   be monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   have ops run this and not block the deploy. Note that while adding an index is a schema
+    #   change, it's completely safe to run the operation after the code has deployed.
+    is_dangerous = True
+
+    # This flag is used to decide whether to run this migration in a transaction or not. Generally
+    # we don't want to run in a transaction here, since for long running operations like data
+    # back-fills this results in us locking an increasing number of rows until we finally commit.
+    atomic = False
+
+    dependencies = [
+        ("sentry", "0266_add_dashboard_widget_detail_field"),
+    ]
+
+    operations = [
+        migrations.AddIndex(
+            model_name="release",
+            index=models.Index(
+                fields=["organization", "version"],
+                name="version_btree_idx",
```

Comment:
Let's call this `sentry_release_version_btree_idx`

Reply:
Actually I remember why I wasn't able to do this 
`['Index names cannot be longer than 30 characters.']`